Docs/articles/Kernel/Startup.md

```diff
@@ -1,3 +1,5 @@
 On startup, there is some hand-coded assembly that runs before the Cosmos layer kicks in. From there, the C# entry point Cosmos.System.Kernel.Start() is called.
 
 Cosmos.System.Kernel is an abstract class that forms the Cosmos framework upon which your OS is built upon.
+
+You Can override the more method to add your code When Kernel.Start() is called.
```

Comment:
What does it mean? we have a method called "more"?
Also - fix capitals.

Reply:
I Forgat to pay attention on that i fix it